src/tests/units/controllers/AuthController.test.js

```diff
@@ -0,0 +1,142 @@
+// /* eslint-disable no-unused-vars */
+// /* eslint-disable no-underscore-dangle */
+// import { expect } from 'chai';
+// import app from '../app';
+//
+// const agent = require('supertest');
+//
+// const prefix = '/api/v1/auth';
+//
+// describe('Social authentication', () => {
+//   const prepare = () => {
+//     const httpServer = app.listen();
+//     const request = agent(app);
+//     return {
+//       request,
+//       httpServer
+//     };
+//   };
+//
+//   const createAuthenticatedFacebookUser = (done) => {
+//     const { request, httpServer } = prepare();
+//     request.get(`${prefix}/facebook/callback`)
+//       .end((error, resp) => {
+//         done(request);
+//         httpServer.close();
+//       });
+//   };
+//
+//   const createAuthenticatedGoogleUser = (done) => {
+//     const { request, httpServer } = prepare();
+//     request.get(`${prefix}/google/callback`)
+//       .end((error, resp) => {
+//         done(request);
+//         httpServer.close();
+//       });
+//   };
+//
+//   it('should work with Facebook Authentication', done => {
+//     createAuthenticatedFacebookUser(req => {
+//       req.get(`${prefix}/access`)
+//         .end((err, res) => {
+//           expect(res.status).eql(201);
+//           done(err);
+//         });
+//     });
+//   });
+//
+//   it('should work with Google Authentication', done => {
+//     createAuthenticatedGoogleUser(req => {
+//       req.get(`${prefix}/access`)
+//         .end((err, res) => {
+//           expect(res.status).eql(201);
+//           done(err);
+//         });
+//     });
+//   });
+//
+//   // it('should deny access to /api/v1/auth/access', done => {
+//   //   const { request, httpServer } = prepare();
+//   //   request.get(`${prefix}/access`)
+//   //     .end((err, res) => {
+//   //       expect(res.status).eql(403);
+//   //       httpServer.close();
+//   //       done(err);
+//   //     });
+//   // });
+// });
+/* eslint-disable no-console */
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable no-unused-vars */
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable import/named */
+/* eslint-disable import/no-named-as-default */
+/* eslint-disable import/no-named-as-default-member */
```

Comment:
Great work @gildniy but I think you would remove these comments

Reply:
Great work! I think this PR will be ready to be merged once you remove these comments.

Reply:
Ok thanks, let me do that.